vrs/marten-construction-script.py

```diff
@@ -180,12 +161,12 @@ def main(args):
     ht_union = hts[0].union(*hts[1:])
 
     logger.info(
-        f"Total size of random, indel, long ref, long var, sex chr combined Table as: %i",
+        f"Total size of random, indel, long ref, long var, sex chr combined Table as: %d",
```

Comment:
i changed the wording because usually interpret "size" to mean size of the file itself